org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_manage-groups.scss

```diff
@@ -132,55 +126,51 @@
       }
 
       .crm-accordion-body {
-        padding: 15px 0 !important;
+        padding: 15px 0 0 !important;
 
         .section-shown {
           padding: 0;
         }
       }
     }
 
-    .crm-action-menu .select2-default {      
-      span.select2-chosen {
-        color: $crm-copy;
+    .crm-action-menu .select2-default span.select2-chosen {
+      color: $crm-copy;
 
-        .crm-i {
-          display: none;
-        }
+      .crm-i {
+        display: none;
       }
     } 
 
     .description {
-      margin: 15px;
       font-family: $font-family-base;
+      margin: 15px;      
     }
 
     #_qf_GroupContact_next {
+      background: $brand-primary url(../../../../../i/plus-icon.png) 8px 11px no-repeat;
```

Comment:
Well, i believe this should be changed. I can see that you are trying to get image from outside the extension itself "org.civicrm.bootstrapcivicrm". Images should be come from the same extension and shouln't rely on any other extension. Please try to use fontawesome if possible or create image directory in the same extension and put ur images inside it. I do recommend the font awesome solution if possible